tgui/packages/tgui/interfaces/Sentencing.js

```diff
@@ -0,0 +1,405 @@
+import { Flex, LabeledList, Box, Section, Tabs, Button } from '../components';
+import { Window } from '../layouts';
+
+const PAGES = {
+  'main': () => MainMenu,
+  'import_report': () => ImportReport,
+  'incident_report': () => NewReport,
+  'new_charge': () => NewCharge,
+};
+
+export const Sentencing = (props, context) => {
+  const { data } = useBackend(context);
+  const { current_menu } = data;
+  const PageComponent = PAGES[current_menu]();
+
+  return (
+    <Window theme="weyland" width={780} height={725}>
+      <Window.Content>
+        <PageComponent />
+      </Window.Content>
+    </Window>
+  );
+};
+
+const MainMenu = (props, context) => {
+  const { act } = useBackend(context);
+
+  return (
+    <Flex
+      direction="column"
+      justify="center"
+      align="center"
+      height="100%"
+      fontFamily="monospace"
+      color="darkgrey"
+      fontSize="2rem"
+      mt="-4rem"
+      bold>
+      <Box>Jurisdictional Automated System</Box>
+      <Box mb="2rem">WY-DOS Executive</Box>
+      <Box>Version 5.8.4</Box>
+      <Box>Copyright © 2182, Weyland Yutani Corp.</Box>
+
+      <Flex mt="5rem">
+        <Button
+          content="Import Incident"
+          width="30vw"
+          height="100%"
+          textAlign="center"
+          fontSize="1.5rem"
+          p="1rem"
+          mr="2rem"
+          onClick={() => act('set_menu', { new_menu: 'import_report' })}
+        />
+
+        <Button
+          content="New Report"
+          width="30vw"
+          height="100%"
+          textAlign="center"
+          fontSize="1.5rem"
+          p="1rem"
+          onClick={() => act('new_report')}
+        />
+      </Flex>
+    </Flex>
+  );
+};
+
+const ImportReport = (props, context) => {
+  const { act } = useBackend(context);
+
+  return (
+    <Flex
+      direction="column"
+      justify="center"
+      align="center"
+      height="100%"
+      fontFamily="monospace"
+      color="darkgrey"
+      fontSize="2rem"
+      mt="-4rem"
+      bold>
+      <Box>
+        XS-2 Scanner:{' '}
+        <Box inline color="green">
+          Online
+        </Box>
+      </Box>
+      <Box fontSize="1.5rem" my="2rem">
+        Insert incident report paper
+      </Box>
+      <Button
+        content="Cancel"
+        textAlign="center"
+        fontSize="1.5rem"
+        width="30vw"
+        p="1rem"
+        onClick={() => act('set_menu', { new_menu: 'main' })}
+      />
+    </Flex>
+  );
+};
+
+const NewReport = (props, context) => {
+  const { data, act } = useBackend(context);
+  const { suspect_name, summary, sentence, current_charges } = data;
+
+  const canExport = suspect_name && current_charges.length;
+
+  return (
+    <>
+      <Flex mb=".25rem" align="center">
+        <Button.Confirm
+          icon="arrow-left"
+          px="2rem"
+          textAlign="center"
+          mr="1rem"
+          onClick={() => act('set_menu', { new_menu: 'main' })}
+        />
```

Comment:
this seemingly sends the paper inside into nullspace and makes it inaccessible - you should properly get rid of the current incident / export it / explain it is being deleted when you do this